packages/conform-dom/intent.ts

```diff
@@ -21,12 +21,7 @@ type ExtractListIntentPayload<Operation, Schema = unknown> = Pretty<
 	>
 >;
 
-/**
- * Helpers to configure an intent button for modifying a list
- *
- * @see https://conform.guide/api/react#list
- */
-export const list = new Proxy<{
+type List = {
```

Comment:
the idea here is this is used as our type contract, while the implementation is casted where needed.